scripts/r.in.wms/wms_drv.py

```diff
@@ -18,35 +18,32 @@
 """
 
 import socket
-import grass.script as gs
-
 from time import sleep
 
+import grass.script as gs
+
 try:
     from osgeo import gdal
-except:
+except ImportError:
     gs.fatal(
         _(
             "Unable to load GDAL Python bindings (requires package 'python-gdal' "
             "being installed)"
         )
     )
 
-import numpy as np
-
-np.arrayrange = np.arange
-
-from math import pi, floor
-
-from urllib.error import HTTPError
 from http.client import HTTPException
-
+from math import floor, pi
+from urllib.error import HTTPError
 from xml.etree.ElementTree import ParseError
 
-from wms_base import GetEpsg, GetSRSParamVal, WMSBase
+import numpy as np
 
-from wms_cap_parsers import WMTSCapabilitiesTree, OnEarthCapabilitiesTree
 from srs import Srs
+from wms_base import GetEpsg, GetSRSParamVal, WMSBase
+from wms_cap_parsers import OnEarthCapabilitiesTree, WMTSCapabilitiesTree
+
+np.arrayrange = np.arange
```

Comment:
Would you mind creating a follow up PR changing usages of np.arrayrange with np.arange? I never knew about it in the last 10 years. It's been deprecated for so long that I can barely find docs for it, the only traces I find is old copies of numpy source code that was used to deprecate it.

Reply:
Got it, will do another PR updating this once this is merged